app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -280,9 +292,10 @@ class RoutePresenter(
         activity.startActivity(browserIntent, null)
     }
 
+    @RequiresApi(Build.VERSION_CODES.O)
```

Comment:
Aren't there other `openSetting` route actions that don't require Oreo? IIRC the `RouteAction.SystemSetting.Security` also goes through this method, but doesn't have a minversion.